src/kmeds_pywrapper.cpp

```diff
@@ -35,11 +35,9 @@ class KMedsWrapper : public KMedoids {
    *
    * @param inputData Input data to find the medoids of
    * @param loss The loss function used during medoid computation
-   * @param k The number of medoids to compute
    * @param logFilename The name of the outputted log file
    */
-  void fitPython(py::array_t<double> inputData, std::string loss, int k, std::string logFilename) {
-    KMedoids::setNMedoids(k);
+  void fitPython(py::array_t<double> inputData, std::string loss, std::string logFilename) {
```

Comment:
Ah, I think this is the wrong thing to do. We *want* the user to be able to call `.fit()` from Python with a specific number of medoids, or instantiate the `KMedoids` object with a value for `k`. 

#48  was actually just referring to changing the `README` documentation -- not the C++ or Python code, just the example that uses `k=3` in two places instead of one

Reply:
Oh, it make senses, I think I have overthought the issue!